lib/samples/show_service_area/show_service_area.dart

```diff
@@ -260,20 +253,7 @@ class _ShowServiceAreaState extends State<ShowServiceArea>
       // Re-enable the UI once the service area task is finished.
       setState(() => _taskInProgress = false);
     } else {
-      showDialog(
-        context: context,
-        builder: (context) => AlertDialog(
-          content: const Text(
-            'At least 1 facility is required to perform a service area calculation.',
-          ),
-          actions: [
-            TextButton(
-              onPressed: () => Navigator.pop(context, 'OK'),
-              child: const Text('OK'),
-            ),
-          ],
-        ),
-      );
+      showMessageDialog('At least 1 facility is required to perform a service area calculation.', showOK: true,);
```

Comment:
run `dart format .` to get the desired formatting (here and probably elsewhere too)